web/src/components/AdminCard.module.css

```diff
@@ -0,0 +1,202 @@
+.container {
+    display: inline-block;
+    position: relative;
+    width: 300px;
+    height: 80px;
+}
+
+.adminName {
+    composes: body3 from '../App.css';
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.primaryAdminName {
+    composes: caption1 from '../App.css';
+    white-space: nowrap;
+}
+
+.adminCard {
+    width: 100%;
+    height: 100%;
+    border-radius: 5px;
+    border: 1px solid var(--dark-grey);
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 16px 0 16px;
+    text-overflow: ellipsis;
+}
+
+.deleteIcon {
+    width: 26px;
+    height: 26px;
+    margin-left: 48px;
+    cursor: pointer;
+    position: absolute;
+    top: 35%;
+    left: 100%;
+}
+
+.lockIcon {
+    width: 31px;
+    height: 36px;
+    margin-left: 45px;
+    position: absolute;
+    top: 30%;
+    left: 100%;
+}
+
+.modal {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    overflow: auto;
+    backdrop-filter: blur(5px) brightness(68%);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.modalContentManage {
+    padding: 60px;
+    width: 50%;
+    height: 80%;
```

Comment:
Can you remove the height:80% please? It looks funny on my screen but it looks totally fine if we just let it shrinkwrap.